torch/csrc/jit/codegen/cuda/lower_index.h

```diff
@@ -10,30 +10,39 @@ namespace torch {
 namespace jit {
 namespace fuser {
 
-class TORCH_CUDA_API IndexLowering : public OptOutMutator {
+class TORCH_CUDA_API IndexLowering : public OptInDispatch {
  private:
```

Comment:
please use separate private sections for data member and methods (private methods, then private data)